src/insertionsort.c

```diff
@@ -0,0 +1,27 @@
+/** .  
+ * @file insertionsort.c
+ * for doxy
+ * 
+ * 
+ */
+#include <stdio.h>
+#include "../include/search_index.h"
+
+
+void insertionsort(int **input, int sizeof_array){
+
+    int index = search_index(sizeof_array);
```

Comment:
Das Prinzip ist, dass es ein KEY element gibt. Dieses element ist zu beginn das 2. im Array. Alle Zahlen vor diesem Key werden verglichen, diesmal aber ab dem KEY element reversiv also nach links bis das 0. Element erreicht wurde ODER ein Element das Größer ist als das KEY element, dann wird das KEY element in der Stelle Inserted und alle elemente rutschen um eine Stelle nach rechts